src/components/LanguageSelector.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { PropTypes } from 'react';
+import {
+  MenuItem,
+  SelectField,
+} from 'material-ui';
+
+const LanguageSelector = (props) => {
+  const {
+    language,
+    onChange,
+    style,
+  } = props;
+
+  return (
+    <SelectField
+      disabled
+      id="languageSelector"
+      onChange={onChange}
+      style={style}
+      value={language}
+    >
+      <MenuItem value="python" primaryText="Python 3"/>
```

Comment:
Again, this will work for setting Codemirror mode correctly, but if we want to use this as a variable to retrieve a parser, I'd rather have to shave off the `3` for Codemirror than clear up an ambiguous call to the parser.